packages/sites/ortho-site/package.json

```diff
@@ -89,5 +89,8 @@
   "files": [
     "dist",
     "webapp"
-  ]
+  ],
+  "dependencies": {
+    "patristic": "^0.6.0"
```

Comment:
I think this should be declared in packages/libs/components/package.json.

Reply:
There's
```
import { parseNewick } from 'patristic';
```
in Sequences.tsx though.

Reply:
We don't use patristic directly in `components`. If `tidytree` needs it then that's its business!

Reply:
Aha, thanks!